client/my-sites/add-ons/hooks/use-add-ons.ts

```diff
@@ -264,6 +331,43 @@ const useAddOns = ( siteId?: number, isInSignup = false ): ( AddOnMeta | null )[
 					description,
 				};
 			} );
+	},
+	(
+		state,
+		activeAddOns,
+		spaceUpgradesPurchased,
+		siteId,
+		isLoadingBillingTransactions,
+		mediaStorage
+	) => [
+		getFeaturesBySiteId( state, siteId ),
+		getSiteOption( state, siteId, 'is_commercial' ),
+		state.productsList,
+		activeAddOns,
+		spaceUpgradesPurchased,
+		siteId,
+		isLoadingBillingTransactions,
+		mediaStorage,
+	]
```

Comment:
Curious if this is correct and comprehensive enough 🤔 
We want a memorized selector given all the state pieces used in the body and the parameters passed to the selector

Reply:
Great question. I'm actually thinking whether it's too comprehensive . For add-ons, my intuition is that `activeAddOns`, `spaceUpgradesPurchased`, `siteId`, `isLoadingBillingTransactions` and `mediaStorage` should be enough. However, it doesn't hurt to keep it as it is now since the other things enlisted are also quite cheap :)